dashboard/config/locales.yml

```diff
@@ -191,6 +191,11 @@
   english: "Indonesian"
   native: "Bahasa Indonesia"
 
+"in": "in-TL"
+"in-TL":
+  english: "Crowdin In-Context"
+  native: "Crowdin In-Context"
+
```

Comment:
This should probably have the "debug" attribute added to it so it doesn't show up for normal students. See the other configurations as an example.